src/components/JsonSchemaValidator/dbtFractribution.js

```diff
@@ -0,0 +1,120 @@
+export const dbtSnowplowFractributionConfigSchema = {
+  type: 'object',
+  properties: {
+    snowplow__page_views_source: {
+      type: 'string',
+      title: 'Page Views Source',
+      description: 'Source of the snowplow_web_page_views table',
+    },
+    snowplow__web_user_mapping_table: {
+      type: 'string',
+      title: 'User Mapping Table',
+      description: 'Schema and table of the User Mapping table',
+    },
+    snowplow__conversions_source: {
+      type: 'string',
+      title: 'Conversions Source',
+      description: 'Source of your conversions table',
+    },
+    snowplow__conversion_window_start_date: {
+      type: 'string',
+      format: 'date',
+      title: 'Conversions Window Start Date',
+      description: 'Start date (UTC) of your conversion window',
+    },
+    snowplow__conversion_window_end_date: {
+      type: 'string',
+      format: 'date',
+      title: 'Conversions Window End Date',
+      description: 'End date (UTC) of your conversion window',
+    },
+    snowplow__path_lookback_days: {
+      type: 'integer',
+      title: 'Path Lookback (days)',
+      description:
+        'Days to lookback from the conversion to include in the path',
+      minimum: 1,
+    },
+    snowplow__path_lookback_steps: {
+      type: 'integer',
+      title: 'Path Lookback Steps',
+      description:
+        'Maximum number of steps before the conversion (0 is unlimited)',
+      minimum: 0,
+    },
+    snowplow__path_transforms: {
```

Comment:
I know how the frac package works with the path transforms and I still got confused by this section. i'm not sure how difficult it is to add some instruction under the snowplow__path_transforms heading, but I think we need to give some more instruction. I understand why, but it's especially confusing having some be dropdowns with null as the option and some text entry without much info as to why unless you go trawl through the udfs.sql file